orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -467,6 +467,38 @@ class SqlExecutionRepository(
         .fetch(0, String::class.java)
     }
 
+  override fun retrievePipelineConfigIdsForApplicationWithCriteria(
+    application: String,
+    criteria: ExecutionCriteria
+  ): List<String> {
+    var baseQueryPredicate = field("application").eq(application)
+    var table = if (jooq.dialect() == SQLDialect.MYSQL) PIPELINE.tableName.forceIndex("pipeline_application_idx")
```

Comment:
As a rule, try to avoid force indexes - let the database optimize as needed as if new indexes come online or old ones get removed this would require work here.

Reply:
Following the optimised retrieval from this PR https://github.com/spinnaker/orca/pull/4804 I can revert to a simple query. What do you think?

Reply:
Please!  Simple == better on these let the DB do it's thing... particularly if in future indexes change.